### ISSUE TITLE
'answers.microsoft.com' & 'docs.microsoft.com' account (sign-in) problem

### ISSUE THREAD

**Issue by markfilipak-windows** [MicrosoftDocs/feedback#889]
I go here:
https://answers.microsoft.com/en-us/
and sign in. Then I go here:
https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/clean-up-the-winsxs-folder
and see in the page's upper right, "Sign in" (as though I'm not signed in).
When I click the "Sign in" link, I'm taken to this registration page (as though I don't have an account):
https://techprofile.microsoft.com/en-US/register?replyUrl=https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/clean-up-the-winsxs-folder&

**Comment by markfilipak-windows** [comment]
More...
I signed out at 'answers.microsoft.com'
I went here:
https://www.microsoft.com/en-us/
I signed in. Then I go here:
https://answers.microsoft.com/en-us?auth=1
I'm still signed in. Then I went here:
https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/clean-up-the-winsxs-folder
That page doesn't know I'm signed in.

**Comment by Powerhelmsman** [comment]
@Mei-Liang Can you take a look at this issue?

**Comment by Powerhelmsman** [comment]
@saldana This issue is assigned to you.

**Comment by markfilipak-windows** [comment]
Thanks, Jian.
Sandra, I'll do whatever is needed to help. Also, in the last few days, I've lost the ability to post a new question to 'answers.microsoft.com'. I don't know whether that's related. I didn't mention it in the thread starter because I wanted to post a clean set of symptoms.

**Comment by saldana** [comment]
@markfilipak-windows, sorry for my late response. I was on vacation until today. Unfortunately we are using two different profiles in answers.microsoft.com and in docs.microsoft.com so right now we require to create two accounts. We are aware that this is far from ideal from a user's perspective and we are looking into solving the problem. We do not have a timeline yet for it. I am very sorry about the trouble we are causing you.

**Comment by markfilipak-windows** [comment]
Hello Sandra,

Suggestion: In the interim, can you post a note on the sign-ups that 
inform users of this issue so that they don't think they've done 
something wrong or that something is broken.

Regards,
Mark.
---
----
When you don't know what you want, finding it brings surprise.

**Comment by calia1120** [comment]
@markfilipak-windows @saldana this seems similar to my issue #901

**Comment by markfilipak-windows** [comment]
Yes, it is very similar. You are lucky in a way. When I tried to sign in after SSO was instituted (only partly, it seems), I was prompted to create a nickname. I tried to enter 'markfilipak' and was told it was already taken. Only then did I discover that I was not in my account. When I tried to get into my account, I was refused. I had to create another account as 'markfilipak-' in order to get logged in. I've never been able to get back into 'markfilipak'.

**Comment by LuigiBruno** [comment]
Looks like I'm experiencing the same issue: even if I'm already signed in with my Microsoft account and able to contribute to Forums, I'm always prompted for creating a profile (but I've already have a profile and I've also started attending courses on Microsoft Learn).

**Comment by LuigiBruno** [comment]
I've also tried using different browsers and InPrivate or Incognito browsing: always the same problem, I'm always prompted to register and create a profile.

**Comment by saldana** [comment]
Tech Profile is not currently linked to any other property beyond MS Learn. We will look at adding a message in registration and edit profile page while we work on the long-term solution. Thanks all for the suggestions. I will keep this issue opened for now.

**Comment by saldana** [comment]
@jialge assigning this to you as you will own Tech Profile now while Hui is out.

**Comment by MIMostafa** [comment]
’m having a similar issue. Every time I go to Microsoft Learn and try to create a tech profile, I click on Register and nothing happens. It literally stays on the same page. I tried lots of things but I’m stuck on create profile. For example I intentionally try to put a too short url, I get the error it is too short. Once I put a long enough URL, error disappears but still unable to move past the register stage.

Can you help please??

**Comment by MIMostafa** [comment]
Just found out that it doesn’t work when country is United Kingdom. I tried IS as country and I managed to create a profile!

**Comment by rootsmusic** [comment]
> @jialge assigning this to you as you will own Tech Profile now while Hui is out.

@saldana He isn't assigned.

**Comment by saldana** [comment]
@sharriexie I am assigning this to you. Can you please address this? Thanks!
